tests/test_transcription.py

```diff
@@ -0,0 +1,90 @@
+# CREATED: 2/9/16 2:27 PM by Justin Salamon <justin.salamon@nyu.edu>
+
+import mir_eval
+import numpy as np
+
+A_TOL = 1e-12
+
+
+def _load_unit_test_reference():
```

Comment:
:crown: No worries your grace :crown:  
Just to clarify: you want these merged into a single "load" function, or removed completely and inlined into every function they're called in (there are 2)?